ground/gcs/src/plugins/config/autotuneshareform.cpp

```diff
@@ -0,0 +1,178 @@
+/**
+ ******************************************************************************
+ * @file       autotuneshareform.h
+ * @author     dRonin, http://dronin.org, Copyright (C) 2015
+ * @author     Tau Labs, http://taulabs.org, Copyright (C) 2014
```

Comment:
It's actually an old file moved because I think it's more appropriate here. Perhaps git lost the ref during rebase. :-(